src/drivers/imu/mpu6000/mpu6000.h

```diff
@@ -114,13 +114,13 @@
 #define MPUREG_TRIM2			0x0E
 #define MPUREG_TRIM3			0x0F
 #define MPUREG_TRIM4			0x10
-#define MPU_GYRO_DLPF_CFG_256HZ_NOLPF2	0x00
-#define MPU_GYRO_DLPF_CFG_188HZ		0x01
-#define MPU_GYRO_DLPF_CFG_98HZ		0x02
-#define MPU_GYRO_DLPF_CFG_42HZ		0x03
-#define MPU_GYRO_DLPF_CFG_20HZ		0x04
-#define MPU_GYRO_DLPF_CFG_10HZ		0x05
-#define MPU_GYRO_DLPF_CFG_5HZ		0x06
+#define MPU_GYRO_DLPF_CFG_256HZ_NOLPF2	0x00  // delay: 0.98ms
+#define MPU_GYRO_DLPF_CFG_188HZ	0x01  // delay: 1.9ms
+#define MPU_GYRO_DLPF_CFG_98HZ		0x02  // delay: 2.8ms
+#define MPU_GYRO_DLPF_CFG_42HZ		0x03  // delay: 4.8ms
+#define MPU_GYRO_DLPF_CFG_20HZ		0x04  // delay: 8.3ms
+#define MPU_GYRO_DLPF_CFG_10HZ		0x05  // delay: 13.4ms
+#define MPU_GYRO_DLPF_CFG_5HZ		0x06  // delay: 18.6ms
```

Comment:
We should be able to use this delay data to improve IMU timestamping. We should subtract this value from the hrt_absolute_time() we use to timestamp the accel/gyro data.

Reply:
I'm fine with that. You'll have to make sure that it works with the estimators.